scripts/StartOakestraCluster.sh

```diff
@@ -112,13 +112,13 @@ if [ -z "$SYSTEM_MANAGER_URL" ]; then
     exit 1
 fi
 
-rm -rf ~/oakestra 2> /dev/null
-mkdir ~/oakestra 2> /dev/null
+rm -rf ~/oakestra/cluster_orchestrator 2> /dev/null
+mkdir -p ~/oakestra/cluster_orchestrator 2> /dev/null
 
-cd ~/oakestra 2> /dev/null
+cd ~/oakestra/cluster_orchestrator 2> /dev/null
 
 curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/scripts/utils/downloadConfigFiles.sh > downloadConfigFiles.sh
-curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/run-a-cluster/cluster-orchestrator.yml > cluster-orchestrator.yml
+curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/cluster_orchestrator/docker-compose.yml > cluster-orchestrator.yml
```

Comment:
this only works if people actually clone the Oakestra repo. 

`cluster_orchestrator/docker-compose.yml` manually builds the services.
while
`run-a-cluster/cluster-orchestrator.yml` should use the pre-built images.

With this script we don't want the user to clone Oak repo but this should run directly.